python/unitytrainers/trainer.py

```diff
@@ -135,16 +135,18 @@ def write_summary(self, lesson_number):
         Saves training statistics to Tensorboard.
         :param lesson_number: The lesson the trainer is at.
         """
-        if (self.get_step % self.trainer_parameters['summary_freq'] == 0 and self.get_step != 0 and
-                self.is_training and self.get_step <= self.get_max_steps):
+        if self.get_step % self.trainer_parameters['summary_freq'] == 0 and self.get_step != 0:
+            training = self.is_training and self.get_step <= self.get_max_steps
```

Comment:
I'd recommend a more descriptive variable name - we have `training` and `is_training`.